distribution/docker/solr/README.md

```diff
@@ -0,0 +1,32 @@
+# DDF Compatible Solr Container
```

Comment:
I feel like the definition Solr container should be generic enough (and stable enough) to make it a Codice project outside of DDF.  In fact, I would strongly support a ticket that pulls the creation of the solr-distro.zip file out of DDF and moving it, along with the ability to containerize it, into a new Codice repo.

(Would probably be done as part of another PR)

Reply:
Yeah I was actually talking about exactly that with @lessarderic earlier since it seems like that would be the best approach. I think @pklinef had some concerns about that however since we would (in order to make dependencies flow only in a single direction) need to move the solr schemas out and probably any code that is affected by changes to those schemas

Reply:
The other thing I'm not sure on is how we would handle versioning of the solr-distro stuff. Would we have the separate repo just use the same version number as the target ddf version (probably simplest, but feels slightly odd) or would we have it use the solr version that it is building on top of?

Another option could be a combo of the two where it would produce a version like <target_ddf_version>-<solr-version>.

Additionally it could simply start off with a completely new version number if all the solr related code lived in the new repo. With this approach consuming projects would manage updating the version of ddf-solr (for lack of a better name at this time) as needed.